resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -205,7 +218,7 @@ private[spark] object Config extends Logging {
       .createWithDefault(0.1)
 
   val PYSPARK_MAJOR_PYTHON_VERSION =
-    ConfigBuilder("spark.kubernetes.pyspark.pythonversion")
+    ConfigBuilder("spark.kubernetes.pyspark.pythonVersion")
```

Comment:
nit / curiousity: why are we changing this?

Reply:
idiomatic for configs to have camelCase.

Reply:
K

Reply:
We _might_ want to make that (small) change in a separate PR just so that if this slips and doesn't make the cut for 2.4 we aren't trying to change a config string later.

Reply:
hmm good call.... I might do that then